test/end-to-end/thirdpartyemailpassword.test.js

```diff
@@ -141,6 +144,28 @@ describe("SuperTokens Third Party Email Password", function () {
             assert.deepStrictEqual(pathname, "/dashboard");
         });
 
+        it("should show error message on sign up with duplicate email", async function () {
+            await toggleSignInSignUp(page);
+
+            // Clear cookies, try to signup with same address.
+            consoleLogs = await clearBrowserCookiesWithoutAffectingConsole(page, consoleLogs);
+            await page.goto(`${TEST_CLIENT_BASE_URL}/auth`, {
+                waitUntil: "networkidle0",
+            });
+            await toggleSignInSignUp(page);
+
+            // Set values.
+            const [, hasEmailExistMethodBeenCalled] = await Promise.all([
+                setInputValues(page, [{ name: "email", value: "john.doe@supertokens.io" }]),
+                hasMethodBeenCalled(page, EMAIL_EXISTS_API),
+            ]);
+
+            // Assert.
+            assert.strictEqual(hasEmailExistMethodBeenCalled, false);
+            let formFieldErrors = await getFieldErrors(page);
+            assert.deepStrictEqual(formFieldErrors, ["This email already exists. Please sign in instead"]);
+        });
+
```

Comment:
missing test for emailpassword recipe

Reply:
i think it is already there in `signup.test.js` file